DOCKER.md

```diff
@@ -132,29 +125,46 @@ This configuration assumes:
 
 Replace `${HOME}/bundles` with the actual path to your bundles directory if needed.
 
-## Usage Examples
+## Using the MCP Inspector
 
-### Initialize a Bundle
+For interactive testing and exploration of the MCP server, we recommend using the [MCP Inspector](https://github.com/modelcontextprotocol/inspector), which provides a graphical interface for interacting with MCP servers.
 
-```bash
-# Using echo to send a request to initialize a bundle
-echo '{"jsonrpc":"2.0","id":"1","method":"call_tool","params":{"name":"initialize_bundle","arguments":{"source":"/data/bundles/bundle.tar.gz"}}}' | ./scripts/run.sh
-```
+### Setting up the MCP Inspector
 
-### Execute kubectl Commands
+1. Clone the Inspector repository:
 
 ```bash
-# Using echo to send a request to execute a kubectl command
-echo '{"jsonrpc":"2.0","id":"1","method":"call_tool","params":{"name":"kubectl","arguments":{"command":"get pods"}}}' | ./scripts/run.sh
+git clone https://github.com/modelcontextprotocol/inspector.git
+cd inspector
 ```
 
-### Explore Files
+2. Build and run the Inspector:
 
 ```bash
-# Using echo to send a request to list files
-echo '{"jsonrpc":"2.0","id":"1","method":"call_tool","params":{"name":"list_files","arguments":{"path":"/"}}}' | ./scripts/run.sh
+npm install
+npm start
 ```
```

Comment:
We should just tell people to use npx like this `npx @modelcontextprotocol/inspector` and not tell them how to download and compile.